cdap-master/src/test/java/co/cask/cdap/master/environment/k8s/MasterServiceMainTestBase.java

```diff
@@ -128,8 +128,7 @@ public static void finish() {
     );
     DatasetDefinition tableDef = injector.getInstance(Key.get(DatasetDefinition.class,
                                                               Names.named(Constants.Dataset.TABLE_TYPE)));
-    StructuredTableRegistry tableRegistry = new NoSqlStructuredTableRegistry(tableDef,
-                                                                             new ConstantTransactionSystemClient());
+    StructuredTableRegistry tableRegistry = new NoSqlStructuredTableRegistry(tableDef);
```

Comment:
This change is needed, due to the change in https://github.com/cdapio/cdap/commit/3eeccc67cc55e0d2eb94823c6d4cfd923a7eaf93#diff-3db9b15932ab9b8667110079c9772f01R68.